modin/utils.py

```diff
@@ -606,6 +606,46 @@ def try_cast_to_pandas(obj: Any, squeeze: bool = False) -> Any:
     return obj
 
 
+def trigger_import(obj: Any) -> None:
```

Comment:
> I feel like we need to notify user if this trigger failed (due to old HDK for instance).

I believe there will be an exception from HDK in this case. It's enough?

Reply:
Your questions got me thinking about whether data trigger functionality is needed. Shouldn't we do this always when we use the functionality to materialize all computations?

It’s also interesting to know @AndreyPavlenko opinion on this matter.

Reply:
I mean, should we **always** trigger import data on HDK? (not by request)

Reply:
No. The data is only imported before the execution on the HDK side. Some operations could be performed with arrow, in this case, the data is not imported. The arrow execution is only performed if we have an arrow table in partitions. When we force import, the arrow table is imported to HDK. After that, we don't have the arrow table anymore and, thus, the subsequent execution to be performed with HDK. In benchmarks, force import is used to separate the data load and the execution time counting.

Reply:
> In benchmarks, force import is used to separate the data load and the execution time counting.

But how can we make fair measurements in this case? In one case, we know that the next operation will be performed on HDK, so we import data so as not to measure the time of data movement, but in the other case, not.

Reply:
The import time is measured separately, on the load data stage.

Reply:
I see, thanks

Reply:
> Do we guarantee that reading was executed for modin on ray?

@Egor-Krivov this concept does not exist for any engine other than HDK. Therefore, I removed the separate function and left a specific parameter for this with the mention that it can be used for different engines, but any actions will be performed only for HDK.